src/App/EditPage.xaml

```diff
@@ -31,17 +31,17 @@
                 </Grid.ColumnDefinitions>
                 <TextBlock Grid.Row="0" Grid.ColumnSpan="2" x:Name="EditFlyoutTextHeader" Style="{ThemeResource FlyoutPickerTitleTextBlockStyle}" Padding="5" HorizontalAlignment="Center"/>
                 <TextBlock Grid.Row="1" Grid.Column="0" x:Uid="TaskName" Padding="5"/>
-                <TextBox Grid.Row="1" Grid.Column="1" x:Name="TestNameBox" Text="" MinWidth="400"/>
+                <TextBox Grid.Row="1" Grid.Column="1" x:Name="TestNameBox" Text="Enter Task Name" MinWidth="400" AutomationProperties.Name="Task Name"/>
                 <TextBlock Grid.Row="2" Grid.Column="0"  x:Name="PathBlock" x:Uid="TaskPath" Padding="5"/>
-                <TextBox Grid.Row="2" Grid.Column="1" x:Name="TaskPathBox" Text="" MinWidth="400"/>
+                <TextBox Grid.Row="2" Grid.Column="1" x:Name="TaskPathBox" Text="" MinWidth="400" AutomationProperties.Name="Task Path"/>

```

Comment:
instead of explicitly defining an AutomationProperties.Name, consider using AutomationProperties.LabeledBy, and reference the related TextBlock. EX: https://github.com/microsoft/FactoryOrchestrator/blob/main/src/App/ConsolePage.xaml#L28

The benefit is if you simply change the textblock's text, the automation name is updated too :)

Reply:
Resolved in the following commit.